scripts/build_backend.py

```diff
@@ -40,87 +40,6 @@
     IS_WIN = True
 
 
-def build_cpp(

```

Comment:
Man, I am glad this is deleted. The naming and lack of documentation makes it really hard for me to see how this is sycl related.  Who knows this code the best here?  As it is I would need to really sit down and take time here to give this deletion a faithful review.

Reply:
The reason I removed is because it was only referenced from build_oneapi_backend function in setup.py which is also removed